website/common/locales/en/pets.json

```diff
@@ -1,101 +1,105 @@
+  "useGems": "If you've got your eye on a pet, but can't wait any longer for it to drop, use Gems in <strong>Inventory > Market</strong> to buy one!",
+  "hatchAPot": "Hatch a <%= potion %> <%= egg %>?",
+  "hatchedPet": "You hatched a <%= potion %> <%= egg %>!",
+  "displayNow": "Display Now",
+  "displayLater": "Display Later",
+  "petNotOwned": "You do not own this pet.",
+  "earnedCompanion": "With all your productivity, you've earned a new companion. Feed it to make it grow!",
+  "feedPet": "Feed <%= article %><%= text %> to your <%= name %>?",
+  "useSaddle": "Saddle <%= pet %>?",
+  "raisedPet": "You grew a <%= pet %>!",
+  "earnedSteed": "By completing your tasks, you've earned a faithful steed!",
+  "rideNow": "Ride Now",
+  "rideLater": "Ride Later",
+  "petName": "<%= potion(locale) %> <%= egg(locale) %>",
+  "mountName": "<%= potion(locale) %> <%= mount(locale) %>",
+  "petKeyName": "Key to the Kennels",
+  "petKeyPop": "Let your pets roam free, release them to start their own adventure, and give yourself the thrill of Beast Master once more!",
+  "petKeyBegin": "Key to the Kennels: Experience <%= title %> Once More!",
+  "petKeyInfo": "Miss the thrill of collecting pets? Now you can let them go, and have those drops be meaningful again!",
+  "petKeyInfo2": "Use the Key to the Kennels to reset your non-quest collectible pets and/or mounts to zero. (Quest-only and Rare pets and mounts are not affected.)",
+  "petKeyInfo3": "There are three Keys to the Kennels: Release Pets Only (4 Gems), Release Mounts Only (4 Gems), or Release Both Pets and Mounts (6 Gems). Using a Key lets you stack the Beast Master and Mount Master achievements. The Triad Bingo achievement will only stack if you use the \"Release Both Pets and Mounts\" key and have collected all 90 pets a second time. Show the world just how much of collection master you are! But choose wisely, because once you use a Key and open the kennel or stable doors, you won't be able to get them back without collecting them all again...",
+  "petKeyInfo4": "There are three Keys to the Kennels: Release Pets Only (4 Gems), Release Mounts Only (4 Gems), or Release Both Pets and Mounts. Using a Key lets you stack the Beast Master and Mount Master achievements. The Triad Bingo achievement will only stack if you use the \"Release Both Pets and Mounts\" key and have collected all 90 pets a second time. Show the world just how much of collection master you are! But choose wisely, because once you use a Key and open the kennel or stable doors, you won't be able to get them back without collecting them all again...",
+  "petKeyPets": "Release My Pets",
+  "petKeyMounts": "Release My Mounts",
+  "petKeyBoth": "Release Both",
+  "confirmPetKey": "Are you sure?",
+  "petKeyNeverMind": "Not Yet",
+  "petsReleased": "Pets released.",
```

Comment:
I think you have have change all the line endings here. PLease check that they are not CLFR